webpack/dev-server.js

```diff
@@ -12,7 +12,7 @@ var config = require('./dev.config');
 var compiler = webpack(config.webpack);
 var devServer = new WebpackDevServer(compiler, config.server.options);
 
-devServer.listen(config.server.port, 'localhost', function () {
+devServer.listen(config.server.port, '0.0.0.0', function () {
```

Comment:
no need for this anymore as it can be passed in as a config var https://github.com/Goodly/TextThresher/pull/13